src/functions/guild/createAutomodRule.js

```diff
@@ -1,41 +1,26 @@
-module.exports = async (d) => {
+module.exports = async(d) => {
     const data = d.util.aoiFunc(d);
-    const [guildID, ruleName, triggerType, eventType, actionType, reason, enableRule = true, ...rest] = data.inside.splits;
+    const [guildId = d.message.guild.id, name, triggerType, triggerMetadata, actions, enabled = true, exemptRoles, exemptChannels, reason] = data.inside.splits;
```

Comment:
`guildID` should be more proper in this term since all function are same sensitive type

Reply:
Fixed.